app/helpers/application_helper.rb

```diff
@@ -115,13 +117,11 @@ def role_allows?(**options)
       end
     end
 
-    Rbac.role_allows?(options.merge(:user => User.current_user)) rescue false
+    Rbac.role_allows?(:user => User.current_user, **options) rescue false
   end
 
   module_function :role_allows?
   public :role_allows?
-  alias_method :role_allows, :role_allows?
-  Vmdb::Deprecation.deprecate_methods(self, :role_allows => :role_allows?)
```

Comment:
I'm good with this but are we sure all callers are removed?

Reply:
At a minimum you need to check the other plugins that bring UI:

```
$ be rake update:print_engines | grep ManageIQ
  ManageIQ::Providers::IbmCloud::Engine:
  ManageIQ::Providers::Lenovo::Engine:
  ManageIQ::Providers::Nsxt::Engine:
  ManageIQ::Providers::Nuage::Engine:
  ManageIQ::Providers::Redfish::Engine:
  ManageIQ::UI::Classic::Engine:
```

Reply:
` ag manageiq* "role_allows[^?_]"` shows up empty for me

Reply:
pulled out into https://github.com/ManageIQ/manageiq-ui-classic/pull/7975